app/controllers/users/two_factor_authentication/email_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Users::TwoFactorAuthentication::EmailController < ApplicationController
+  # Skips 2FA so User can set up the totp via QR code
+  skip_before_action :handle_two_factor_authentication
+  # Skips 2FA setup confirmation callback in ApplicationController.
+  skip_before_action :confirm_two_factor_setup
+
+  def show
+    if params[:resend] == "true"
+      render :show_resend
+    else
+      @resent = true if params[:resent] == "true"
+      render :show
+    end
+  end
+
+  def update
+    if params[:resend] == "true"
+      send_email
+      redirect_to "/users/two_factor_authentication/email?resent=true"
+    end
+  end
+
+private
+
+  def send_email
+    # TODO: This will be implemented in future.
```

Comment:
No, because then the current flow for email 2FA will then result in an error, making it useless. It's just there as a placeholder that will be implemented in a subsequent PR, but shouldn't actually break the journey.